_posts/2025-09-05-insights-wordcamp-europe-2025.md

```diff
@@ -9,7 +9,7 @@ excerpt: "One topic that really stands out on WordCamp Europe 2025, is accessibi
 date: 2025-09-05
 ---
 
-In the beginning of June WordCamp Europe happened, the biggest WordPress event in the world where thought leaders, and industry experts share their insights and knowledge. Luckily, sessions from Tracks 1 and 2 are available on WordPress TV for everybody that missed the event, so you can always dive into the valuable insights at a later moment.
+In the beginning of June 2025 WordCamp Europe happened, the biggest WordPress event in the world where thought leaders, and industry experts share their insights and knowledge. Luckily, sessions from Tracks 1 and 2 are available on WordPress TV for everybody that missed the event, so you can always dive into the valuable insights at a later moment.
 
 One topic that really stands out, especially when you are a European entrepreneur, is accessibility. With the European Accessibility Act (EAA) coming into effect in June, it's clear that this is something we all need to pay attention to. While the Act isn’t specific to WordPress, it affects all website owners, especially those running businesses online.
```

Comment:
Change this tense to "With the European Accessibility Act (EAA) having come into effect in June 2025,"